source/simplex/fe_lib.cc

```diff
@@ -327,6 +328,48 @@ namespace Simplex
   }
 
 
+  template <int dim, int spacedim>
+  FiniteElementDomination::Domination
+  FE_DGP<dim, spacedim>::compare_for_domination(
+    const FiniteElement<dim, spacedim> &fe_other,
+    const unsigned int                  codim) const
+  {
+    (void)fe_other;
+    (void)codim;
+
+    Assert((dynamic_cast<const FE_DGQ<dim, spacedim> *>(&fe_other)),
+           ExcNotImplemented());
+    AssertDimension(dim, 2);
+    AssertDimension(this->degree, fe_other.tensor_degree());
+
+    return FiniteElementDomination::either_element_can_dominate;
```

Comment:
I have not looked up the other DG (L2-conforming) elements, but is this the same as for the `FE_DGQ`? To me this looks similar to the `FE_P` case but I don't recall if DG elements have some different more relaxed requirements regarding hp.

Reply:
> but is this the same as for the FE_DGQ?

Yes, it is:

https://github.com/dealii/dealii/blob/662eb95cfecb4f045941ffc67e396a5edfe6b2b7/source/fe/fe_dgq.cc#L627-L636

https://github.com/dealii/dealii/blob/662eb95cfecb4f045941ffc67e396a5edfe6b2b7/source/fe/fe_dgq.cc#L563-L587